File: app/src/main/java/com/nhq/authenticator/data/contract/SitesAvailable.java
package com.nhq.authenticator.data.contract;

import java.util.ArrayList;
import java.util.List;

public class SitesAvailable {
    public static final List<String> sites;

    static {
        sites = new ArrayList<>();
        sites.add("Google");
        sites.add("Microsoft");
        sites.add("Facebook");
        sites.add("Twitter");
    }

    public static List<String> getListSites() {
        return sites;
    }
}

File: app/src/main/java/com/nhq/authenticator/data/contract/SitesAvailable.kt
package com.nhq.authenticator.data.contract

object SitesAvailable {
    @JvmStatic
    val sites = mutableListOf<String>()

    init {
        sites.add("Google")
        sites.add("Microsoft")
        sites.add("Facebook")
        sites.add("Twitter")
    }
}